### ISSUE TITLE
Add `*_id` fields to every related field on every serializer

### ISSUE THREAD

**Issue by mlissner** [freelawproject/courtlistener#3294]
When you pull the cluster, it gives you a URL to the docket. That's annoying to parse. It should give an ID as well.

When you pull an opinion from the API, it gives you a URL to the cluster. That's annoying to parse. it should give you the cluster ID as well.

**Comment by mlissner** [comment]
If we do this, we should update https://github.com/freelawproject/courtlistener/discussions/3293.

**Comment by mlissner** [comment]
I did a bit of work on this today, and the original issue should be fixed in #3307, but we should do this for all of the `HyperlinkedRelatedField`s we have in our API. I started doing that by hand, but it got boring, and that was a clue to me that we should probably be doing this via a mixin or a class instead. 

Currently our serializer class is a modified version of the `HyperlinkedModelSerializer` that adds an `id` field to every serializer. Maybe we can modify our serializer to also include ID fields for all related objects. Or maybe not, but I wasn't able to spend too much time on this.
